src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -76,15 +76,26 @@
       <Line Include="End Sub"/>
       <Line Include="End Module"/>
     </ItemGroup>
+
+    <ItemGroup Condition="'$(Language)'=='F#'">
+      <Line Include="module Program"/>
```

Comment:
This will fail if another module is named `Program`. It would probably be better to do something like 

```fsharp
module ``VSTest-AutoGenerated-Program``
```

It can still have collisions, but way less likely. Also, both the C# and the VB version is decorated with `TestSDKAutoGeneratedCodeAttribute` which is probably a good idea.

Reply:
Good catch - I'll fix that.